docs/src/rules/sort-keys.md

```diff
@@ -89,6 +89,7 @@ The 2nd option is an object which has the following properties.
 * `minKeys` - Specifies the minimum number of keys that an object should have in order for the object's unsorted keys to produce an error. Default is `2`, which means by default all objects with unsorted keys will result in lint errors.
 * `natural` - if `true`, enforce properties to be in natural order. Default is `false`. Natural Order compares strings containing combination of letters and numbers in the way a human being would sort. It basically sorts numerically, instead of sorting alphabetically. So the number 10 comes after the number 3 in Natural Sorting.
 * `allowLineSeparatedGroups` - if `true`, the rule allows to group object keys through line breaks. In other words, a blank line after a property will reset the sorting of keys. Default is `false`.
+* `ignoreComputedKeys` - if `true`, the rule ignores all computed keys and doesn't report unsorted properties separated by them. Default is `false`.
```

Comment:
You [mentioned](https://github.com/eslint/eslint/issues/19153#issuecomment-2489141458) wanting to reset the count when a computed key was found, and it appears that's how this was implemented, so maybe we should mention that here?
```suggestion
* `ignoreComputedKeys` - if `true`, the rule ignores all computed keys and doesn't report unsorted properties separated by them. A computed key will reset the sorting of the following non-computed keys. Default is `false`.
```

Reply:
Yes, but I realized that "reset" might be confusing so I reworded it as "doesn't report unsorted properties separated by them".

Reply:
Yeah, I couldn't quite figure out what that meant. Given that `allowLineSeparatedGroups` already uses the phrase "will reset the sorting of keys", I think it's good to be consistent.

Reply:
Maybe we could put this in parentheses because it explains the same mentioned in the previous sentence?

```suggestion
* `ignoreComputedKeys` - if `true`, the rule ignores all computed keys and doesn't report unsorted properties separated by them (a computed key will reset the sorting of the following non-computed keys). Default is `false`.
```

Reply:
> Given that `allowLineSeparatedGroups` already uses the phrase "will reset the sorting of keys", I think it's good to be consistent.

Ah, I didn't notice that. In that case, I'll commit your suggestion.